deploy/sdk/src/dynamo/sdk/cli/cli.py

```
@@ -25,6 +25,7 @@
 from dynamo.sdk.cli.build import build
 from dynamo.sdk.cli.deployment import app as deployment_app
 from dynamo.sdk.cli.deployment import deploy
+from dynamo.sdk.cli.dynamo_graph import build, get
```

Ruff (0.11.9)

Redefinition of unused `build` from line 25

(F811)